ndmg/graph/gen_graph.py

```diff
@@ -295,8 +301,8 @@ def save_graph_png(self, graphname):
         from sklearn.preprocessing import normalize
         conn_matrix = np.array(nx.to_numpy_matrix(self.g))
         conn_matrix = normalize(conn_matrix)
-        [z_min, z_max] = -np.abs(conn_matrix).max(), np.abs(conn_matrix).max()
-        plot_matrix(conn_matrix, figure=(10, 10), vmax=z_max * 0.5, vmin=z_min * 0.5, auto_fit=True, grid=False,
+        [z_min, z_max] = np.abs(conn_matrix).min(), np.abs(conn_matrix).max()
+        plot_matrix(conn_matrix, figure=(10, 10), vmax=z_max, vmin=z_min * 0.5, auto_fit=True, grid=False,
                     colorbar=False)
```

Comment:
We should probably include a colorbar on the output here, so that we know what our `vmax` and `vmin`s are

Reply:
Could also use graspy's `pass_to_ranks`: https://github.com/neurodata/graspy/blob/master/graspy/utils/ptr.py#L20

followed by graspy's `heatmap`:
https://github.com/neurodata/graspy/blob/master/graspy/plot/plot.py#L145

if we wanted to, that would get things more in line with the viz code that I wrote so that they match up:
https://github.com/neurodata/graphutils/blob/master/graphutils/graph_stats.py#L125

but up to discretion

Reply:
Actually I think it's necessary here to make `vmin` and `vmax` always the same value (e.g., 0 and 1, or whatever else) - otherwise we won't be able to compare multiple graphs with each other. Thoughts?